test/views/duty_deferment_account/DutyDefermentAccountSpec.scala

```diff
@@ -37,24 +37,29 @@ class DutyDefermentAccountSpec extends SpecBase {
 
       shouldDisplayCorrectCommonGuidanceAndText(viewDoc, msg, accountNumber, serviceUnavailableUrl)
       viewDoc.getElementById("request-statement-link").text() must not be empty
+      viewDoc.html().contains(showAllSectionText) mustBe true
```

Comment:
Could you please check against the message key (cf.account.detail.accordion.show-all-sections) instead of the hard coded text? And also could you keep an assertion to  check the count (as 1) for 'Show all sections' when it is present in the view?